Cargo.toml

```diff
@@ -10,6 +10,7 @@ keywords = ["reflection"]
 categories = ["development-tools::procedural-macro-helpers"]
 readme = "README.md"
 edition = "2018"
+build = "build.rs"
```

Comment:
I would prefer for this not to be a build script that everyone downstream needs to run when they compile the crate. Instead, take a look into how the syn repo handles generated sources for things like syn::visit.